### DISCUSSION TITLE
ChatGPT Discussions!

### DISCUSSION THREAD

**Discussion by YangletLiu** [AI4Finance-Foundation/FinGPT#1] [Ideas]
For the maintainers, here are some tips 💡 for getting started with Discussions. We'll leave these in Markdown comments for now, but feel free to take out the comments for all maintainers to see.

**Comment by emusini** [comment]
Hi team! Any news on when the model will be released?

**Reply by Sajidullah9075** [reply]
Hi

**Reply by YangletLiu** [reply]
We have several papers under review. In academia, there are researchers giving us feedback. This is a precious opportunity for us to polishing our projects. Please wait for our updates, which will release one by one, will not take long. Thanks very much!

**Reply by lihongteng** [reply]
我是国内商品期货从业者，和交易者，能够稳定盈利的交易者朋友资源比较多。国内期货交易数据是可获得的，对于量化交易来说，训练成交数据就足够得住交易策略了。我想做出我的贡献，如果有作用的话。
I am a domestic commodity futures practitioner and trader, and there are more friends who can make stable profits. Domestic futures trading data is available, and for quantitative trading, training trading data is sufficient to justify trading strategies. I want to make my contribution, if it makes a difference.

**Reply by YangletLiu** [reply]
Thanks!  It is a very important area and highly aligned with FinGPT's good.  We welcome your contributions!
